Ntuplizer/plugins/VerticesNtuplizer.cc

```diff
@@ -21,6 +21,27 @@ void VerticesNtuplizer::fillBranches( edm::Event const & event, const edm::Event
 
   event.getByToken(vtxToken_, vertices_);
     
-  nBranches_->nPVs = vertices_->size();
+  nBranches_->PV_N = vertices_->size();
+  
+  nBranches_->PV_filter = true;
+  
+  reco::VertexCollection::const_iterator firstGoodVertex = vertices_->end();
+  int firstGoodVertexIdx = 0;
+  for( reco::VertexCollection::const_iterator vtx = vertices_->begin(); vtx != vertices_->end(); ++vtx, ++firstGoodVertexIdx){
+  
+    nBranches_->PV_chi2.push_back(vtx->chi2());
+    nBranches_->PV_ndof.push_back(vtx->ndof());
+    nBranches_->PV_rho.push_back(vtx->position().Rho());
+    nBranches_->PV_z.push_back(vtx->position().Z());
+    
```

Comment:
I don't think you really need to know if each PV is good. You just need to know if there is at least one good. Indeed I added a filter here

https://github.com/jngadiub/EXOVVNtuplizerRunII/blob/CMSSW_7_4_X/Ntuplizer/plugins/VerticesNtuplizer.cc#L45

But I will add a flag for all of them as well in the next iteration.